### PR TITLE
WT-5045 Add more statistics tracking where checkpoint spends time

### PR DESCRIPTION
**Pull request by tammybailey**
Added five new timing statistics to the prepare stage of checkpoint: `txn_checkpoint_prep_running`, `txn_checkpoint_prep_max`, `txn_checkpoint_prep_min`, `txn_checkpoint_prep_recent` and `txn_checkpoint_prep_total`. These prepare statistics were chosen to correspond to the existing timing statistics for (end-to-end) checkpoint, which allows for straightforward comparison and analysis between them.

### CODE REVIEW

File: dist/stat_data.py [hunk 1]
```diff
@@ -565,6 +565,11 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_checkpoint_fsync_post_duration', 'transaction fsync duration for checkpoint after allocating the transaction ID (usecs)', 'no_clear,no_scale'),
     TxnStat('txn_hs_ckpt_duration', 'transaction checkpoint history store file duration (usecs)'),
     TxnStat('txn_checkpoint_generation', 'transaction checkpoint generation', 'no_clear,no_scale'),
+    TxnStat('txn_checkpoint_prep_running', 'transaction checkpoint prepare currently running', 'no_clear,no_scale'),
```
Comment (agorrod) [inline dist/stat_data.py]: Alphabetical ordering nit `txn_checkpoint_prep_running` should go after `txn_checkpoint_prep_recent`

Comment (tammybailey) [reply dist/stat_data.py]: Thanks for catching that! Fixed.

File: src/include/stat.h [hunk 2]
```diff
@@ -686,6 +686,11 @@ struct __wt_connection_stats {
     int64_t txn_checkpoint_time_max;
     int64_t txn_checkpoint_time_min;
     int64_t txn_checkpoint_time_recent;
+    int64_t txn_checkpoint_prep_running;
+    int64_t txn_checkpoint_prep_max;
+    int64_t txn_checkpoint_prep_min;
+    int64_t txn_checkpoint_prep_recent;
+    int64_t txn_checkpoint_prep_total;
```
Comment (sueloverso) [inline src/include/stat.h]: You need to rerun `s_all` to allow all the auto-generated files to be re-alphabetized too.

Comment (tammybailey) [reply src/include/stat.h]: It seems that `s_all` isn't alphabetising correctly - running after the `stat_data.py` reordering it isn't changing any files. @sueloverso is seeing this same behaviour.

**Pull request merged**